pulsar-common/src/main/java/org/apache/pulsar/common/api/Commands.java

```diff
@@ -168,6 +210,21 @@ public static ByteBuf newConnected(int clientProtocolVersion) {
         return res;
     }
 
+    public static ByteBuf newConnecting(String authMethod, byte[] authData) {
+        CommandConnected.Builder connectedBuilder = CommandConnected.newBuilder();
+        connectedBuilder.setServerVersion("Pulsar Server Mutual Authn");
```

Comment:
If this a version why doesn't it include the actual version (like 2.3.0)

Reply:
Followed a way in line 197 above.